addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "<%= htmlComponentName %>",
   "version": "0.0.0",
   "license": "Apache-2.0",
+  "scripts":{
+    "postinstall": "webdriver-manager update"
```

Comment:
We can run it right before protractor instead, as part of the readme instructions. Afterwards, as part of `ng test` if protractor is integrated with it.